### PR TITLE
feat: updated DD RUM settings

### PR DESCRIPTION
**Pull request by SiTaggart**
Description and useful links to discussions / issues / tickets

### Update DD RUM

Get new things and more privacy settings. Set the environment to cypress when running in Cypress so we can filter the metrics out.

### Cookie consent

We still need cookie consent though

## Checklist

- [ ] Work in a "Draft" branch whilst you are getting feedback to reduce Chromatic costs. Once you are ready for approval, convert the PR to "Ready to review"
- [ ] Ensure all `required` checks have passed
- [ ] Icon changes must have a product design review
- [ ] Website content changes must have a product design review
- [ ] At least two engineers have reviewed any code changes
- [ ] At least one engineering lead has reviewed any core package changes or repository infrastructure
- [ ] Website visual regression testing is run by adding `🕵🏻‍♀️ Run website visual regression` label

### CODE REVIEW

**Review by TheSisb — CHANGES_REQUESTED** [review]

File: packages/paste-website/src/components/CookieConsent.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {Alert} from '@twilio-paste/alert';
+import {Button} from '@twilio-paste/button';
+
+export const CookieConsent: React.FC<{onAccept: () => void; onReject: () => void}> = ({onAccept, onReject}) => {
```
[the remark this answers is not in the document]
Comment (SiTaggart) [reply packages/paste-website/src/components/CookieConsent.tsx:6]: I think Alert all in all is the wrong component, not only semantically, but styling wise it does nothing for us. The buttons don't align with the icon either. Gunna go custom

File: packages/paste-website/src/components/CookieConsent.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  return (
+    <Box zIndex="zIndex90" position="fixed" bottom="0" width="100%">
+      <Alert variant="neutral">
+        <Box display="flex" alignItems="center" columnGap="space40">
+          To help us improve this site we use tools that set cookies. Data collected by those tools is annonimized. If
+          you choose to reject the use of cookies on this site, no analytics service will be initialized.
+          <Button variant="secondary" size="small" onClick={onAccept}>
+            Accept cookies
+          </Button>
+          <Button variant="destructive_link" size="small" onClick={onReject}>
```
[the remark this answers is not in the document]
Comment (SiTaggart) [reply packages/paste-website/src/components/CookieConsent.tsx:16]: I started with that funnily enough, but the alignment of buttons to icon in the alert pushed me another way. I think removing the alert will help

File: packages/paste-website/src/components/CookieConsent.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (serifluous) [reply packages/paste-website/src/components/CookieConsent.tsx:6]: +1 to both comments

File: packages/paste-website/src/pages/_app.tsx [hunk 3]
```diff
@@ -7,30 +7,48 @@ import {datadogRum} from '@datadog/browser-rum';
 import {Theme} from '@twilio-paste/theme';
 
 import packageJSON from '../../../paste-core/core-bundle/package.json';
+import {CookieConsent} from '../components/CookieConsent';
 import {useDarkMode} from '../hooks/useDarkMode';
 import * as gtag from '../lib/gtag';
```
[the remark this answers is not in the document]
Comment (serifluous) [reply packages/paste-website/src/pages/_app.tsx:12]: +1, can't click on anything in the topbar and sidebar, but anything in the content area works.

File: packages/paste-theme-designer/pages/_app.tsx [hunk 4]
```diff
@@ -15,9 +15,16 @@ datadogRum.init({
   service: 'paste-remix',
   env: process.env.ENVIRONMENT_CONTEXT,
   version: '1.0.0',
-  sampleRate: 100,
-  trackInteractions: true,
```
Comment (SiTaggart) [inline packages/paste-theme-designer/pages/_app.tsx:19]: Deprecated properties

File: packages/paste-website/src/components/CookieConsent.tsx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+      borderTopStyle="solid"
+      borderTopWidth="borderWidth10"
+      borderTopColor="colorBorderWeaker"
+      paddingY="space60"
+      paddingX="space40"
+      display="flex"
+      justifyContent="space-around"
+    >
+      <Box display="flex" alignItems="center" columnGap="space40" maxWidth="1200px">
+        To enhance our site, we utilize tools that establish cookies. The data gathered by these tools is anonymized. If
```
Comment (TheSisb) [inline packages/paste-website/src/components/CookieConsent.tsx:22]: I liked the old version more, made it more human.

```suggestion
        To help us improve this site we use tools that set cookies. The data gathered by these tools is anonymized. If you reject the use of cookies, no analytics service will be initiated.
```

**Review by TheSisb — COMMENTED** [review]
Small thought on improving the copy, and a console error in incogneto mode

File: packages/paste-website/src/components/CookieConsent.tsx (hunk 5 above)
Comment (serifluous) [reply packages/paste-website/src/components/CookieConsent.tsx:22]: +1 but add a comma: "...improve this site, we use tools"

**Review by serifluous — APPROVED** [review]

**Review by TheSisb — APPROVED** [review]

**Pull request merged**